index.js

```diff
@@ -18,56 +19,81 @@
  * December 2013
  */
 
-module.exports = Lash
-
 function Lash (spec) {
 
-    var self = {}
-    if (!spec) spec = {}
+    var self = {};
+    if (!spec) spec = {};
+
+    self.startTime = null;
+
+    self.startTimer = function () {
+        self.startTime = process.hrtime();
+    };
+
+    self.time = function () {
+        var stackTime = process.hrtime(self.startTime);
+        var numSeconds = stackTime[0] + stackTime[1] / 1e9;
```

Comment:
iunno, process.hrtime() returns a tuple with [seconds, nanoseconds].

I just make it all seconds, but we have a lot of decimal point resolution. Up to you.